js/project-data.js

```diff
@@ -1,38 +1,128 @@
+  company: ['CANOPY', 'Back End Dev', '2015'],
   description: 'A daily selection of privately personalized reads; no accounts or sign-ups require',
-  tecnologies: ['html', 'css', 'javaScript'],
-  image:  'Snapshoot-Portfolio.png',
+  popupdescription: 'Lorem Ipsum is simply dummy text of the printing and or sign-ups requiretypes',
+  technologies: ['html', 'css', 'javaScript'],
+  image:  './img/Snapshoot-project3.png',
+  liveURL: 'https://osvaldobc.github.io/osvaldojose.github.io/',
+  sourceURL: 'https://github.com/OsvaldoBC/portfolio-setup',
 },
   {
-  Name: 'Multi-Post-Stories',
-  Company: 'CANOPY',
-  role:  'Back End Dev',
-  year:  '2015',
+  name: 'Multi-Post-Stories',
+  company: ['CANOPY', 'Back End Dev', '2015'],
   description: 'A daily selection of privately personalized reads; no accounts or sign-ups require',
-  tecnologies: ['html', 'css', 'javaScript'],
-  image:  'Snapshoot-Portfolio.png',
+  popupdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lo',
+  technologies: ['html', 'css', 'javaScript'],
+  image:  './img/Snapshoot-project4.png',
+  liveURL: 'https://osvaldobc.github.io/osvaldojose.github.io/',
+  sourceURL: 'https://github.com/OsvaldoBC/portfolio-setup',
 },
 ];
-console.log(projectData);
+
+// fill dynamic mobile version"
+
+for (let i = 0; i < projectData.length; i++) {
+  console.log(projectData [i]);
+  let card1 = document.createElement('div');
+  card1.classList.add('container1');
+
+  let portfolio = document.createElement('div');
+  portfolio.classList.add('portfolio');
+
+  let contenido1 = document.createElement('div');
+  contenido1.classList.add('contenido1');
+
+  let cardh2 = document.createElement('h2');
+  cardh2.classList.add('tonic');
+  cardh2.textContent = projectData [i].name;
+  contenido1.appendChild(cardh2);
+
+  const buttons = document.createElement('ul');
+  buttons.classList.add('buttons');
+
+  for (let u = 0; u < projectData [i].technologies.length; u += 1) {
+    const tagLi = document.createElement('li');
+    tagLi.classList.add('int-button');
+    tagLi.textContent = projectData [i].technologies[u];
+
+    buttons.appendChild(tagLi);
+  }
+
+  const info = document.createElement('ul');
+  info.classList.add('info');
+
+  for (let u1 = 0; u1 < projectData [i].company.length; u1 += 1) {
+    const tagLi1 = document.createElement('li');
+    tagLi1.classList.add('tonic-detail2');
+    tagLi1.textContent = projectData [i].company[u1];
+    info.appendChild(tagLi1);
+  }
+
+  let cardp = document.createElement('p');
+  cardp.classList.add('project-detail');
+  cardp.textContent = projectData [i].description;
+
+  let image = document.createElement('img');
+  image.src = projectData [i].image;
+  image.alt = 'imagen';
+
+  const button = document.createElement('button');
+  button.classList.add('see');
+  button.setAttribute('type', 'button');
+  const butText = document.createTextNode('See Project');
+  button.appendChild(butText);
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure that your design match that of the Figma template as much as possible
Your design
![Capture](https://user-images.githubusercontent.com/85095702/189269850-d794e22d-4e78-4f39-98b5-513a9200d5e3.PNG)
![Capture 2](https://user-images.githubusercontent.com/85095702/189269865-77d95ac0-b5c7-4b78-bbcd-c279029c106a.PNG)
Figma template
![189239524-6137eeb4-a0f0-4783-8a00-0accaf74ae81](https://user-images.githubusercontent.com/85095702/189269904-7a9df41c-2011-4524-be7b-5209d61fdcba.png)
![189239615-3f957a64-ad11-445e-b509-f7a7d68f6c9e](https://user-images.githubusercontent.com/85095702/189269920-c9f1d2fd-9d6f-4863-95ac-94ecc61988c2.png)